src/modules/auth/auth.js

```diff
@@ -0,0 +1,43 @@
+import AuthApi from './AuthApi';
+
+// Actions
+const REGISTER_REQ = 'auth/REGISTER_REQ';
+const REGISTER_REQ_SUCCESS = 'auth/REGISTER_REQ_SUCCESS';
+const REGISTER_REQ_FAIL = 'auth/REGISTER_REQ_FAIL';
+
+// Initial State
+const INITIAL_STATE = {token: null};
+
+// Reducer
+export default function reducer(state = INITIAL_STATE, action = {}) {
+	switch (action.type) {
+		case REGISTER_REQ_SUCCESS: {
+			const {token} = action.payload;
+			return {
+				...state,
+				token
+			};
+		}
+
+		default:
+			return state;
+	}
+}
+
+// Action Creators
+
+/**
+ * @param {RegisterDTO} registerDto
```

Comment:
Stands for data transfer object. Basically represents a payload sent in a REST request.